src/javascript/app/pages/user/metatrader/metatrader.js

```diff
@@ -17,6 +18,10 @@ const MetaTrader = (() => {
     const getAccountsInfo = MetaTraderConfig.getAccountsInfo;
 
     const onLoad = () => {
+        const current_account = Client.getBrokerCode(Client.getAllLoginids()[0]);
+        if (current_account === 'MF' || Client.isOptionsBlocked()){
+            DerivBanner.onLoad();
+        }
```

Comment:
I think doing something like this, should also yield the same result you want to achieve.

```
const financial_account = ClientBase.getAccountType() === 'financial';   

if (financial_account || Client.isOptionsBlocked()) {            
   DerivBanner.onLoad();        
}
```

Reply:
Replaced logic, based on your suggestion, thanks.